config/mfa.php

```diff
@@ -48,4 +48,15 @@
 
     ],
 
+    'features' => [
+
+        /**
+         * Laravel's about command provides useful information regarding the state of
+         * your Laravel application. If `about_command` is set to true, we will
+         * show useful information about exchange in about command output.
```

Comment:
comment mentioning exchange :)

Reply:
I've applied the relevant changes, the only thing is now it depends on the full `laravel/framework` rather than just `laravel/contracts`. Don't think that's really an issue though. 🤷🏻

Reply:
ah right, because the About extension is not a contract... :( 

Oh well, I am fine with that, when that is actually our dependency